### PR TITLE
Fix creating complement_of! query for single manager_uuid

### PR DESCRIPTION
**Pull request by slemrmartin**
With one manager_uuid it created query like this:

`SELECT *  FROM   (VALUES ('node-uuid'),) AS active_entities_table('source_ref')`
(unwanted comma after VALUES)

cc @agrare @Ladas

### CODE REVIEW

**Comment by slemrmartin** [comment]
@Ladas it means this is not covered by specs, please add it to place where you are testing this

**Comment by Ladas** [comment]
@slemrmartin could you add the spec please? E.g. to retention_strategies_spec. We should probably test for multiple variants of manager ref (e.g. of sizes 0, 1 and 2)

**Comment by Ladas** [comment]
@slemrmartin could you add the spec please? E.g. to retention_strategies_spec. We should probably test for multiple variants of manager ref (e.g. of sizes 0, 1 and 2)

**Comment by Ladas** [comment]
@slemrmartin could you add the spec please? E.g. to retention_strategies_spec. We should probably test for multiple variants of manager ref (e.g. of sizes 0, 1 and 2)

**Comment by Ladas** [comment]
@slemrmartin could you add the spec please? E.g. to retention_strategies_spec. We should probably test for multiple variants of manager ref (e.g. of sizes 0, 1 and 2)

**Comment by Ladas** [comment]
@slemrmartin could you add the spec please? E.g. to retention_strategies_spec. We should probably test for multiple variants of manager ref (e.g. of sizes 0, 1 and 2)

**Comment by Ladas** [comment]
@slemrmartin could you add the spec please? E.g. to retention_strategies_spec. We should probably test for multiple variants of manager ref (e.g. of sizes 0, 1 and 2)

**Comment by Ladas** [comment]
@slemrmartin could you add the spec please? E.g. to retention_strategies_spec. We should probably test for multiple variants of manager ref (e.g. of sizes 0, 1 and 2)

**Comment by Ladas** [comment]
@slemrmartin could you add the spec please? E.g. to retention_strategies_spec. We should probably test for multiple variants of manager uuids (e.g. of sizes 0, 1 and 2)

File: lib/inventory_refresh/save_collection/saver/sql_helper.rb [hunk 1]
```diff
@@ -138,11 +138,17 @@ def active_entities_query(all_attribute_keys_array, manager_uuids)
...
-        values = [first_value, values].join(",")
+
+        if manager_uuids.blank?
+          values = first_value
+        else
+          # Rest of the values, without the type cast
+          values = manager_uuids.map! do |hash|
+            "(#{all_attribute_keys_array.map { |x| quote(connection, hash[x], x, false) }.join(",")})"
+          end.join(",")
+          values = [first_value, values].join(",")
```
Comment (Ladas) [inline lib/inventory_refresh/save_collection/saver/sql_helper.rb]: I'd probably do it here:

```ruby
values = manager_uuids.map! do |hash|
￼     "(#{all_attribute_keys_array.map { |x| quote(connection, hash[x], x, false) }.join(",")})"
￼end.join(",")

values = values.blank? first_value : [first_value, values].join(",")
```

**Pull request merged**